continuous-integration/readme.md

```diff
@@ -69,7 +65,7 @@ An automated build should encompass the following principles:
   - There are several open source code style validation tools available to choose from ([code style checks](https://github.com/checkstyle/checkstyle), [StyleCop](https://en.wikipedia.org/wiki/StyleCop)).
   - Code standards are maintained within a single configuration file. There should be a step in your build pipeline that asserts code in the latest commit conforms to the known style definition.
 - [ ] **Database Schema Automation**
-  - The build process should automate pulling database schemas from the scm repository and applying it to the target environment. This also includes automating the steps to seed that environment with the necessary test data required to run isolated end-to-end automated tests.
+  - The build process should automate pulling database schemas from the git repository and applied to the target environment. This also includes automating the steps to seed that environment with the necessary test data required to run isolated end-to-end automated tests.
```

Comment:
why assume `git`?

Reply:
git is the recommended scm tool as per the playbook [guidance](https://github.com/microsoft/code-with-engineering-playbook/tree/master/source-control).